src/Akka.Hosting/Coordination/LeaseOptionBase.cs

```diff
@@ -0,0 +1,18 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="LeaseOptionBase.cs" company="Akka.NET Project">
+//      Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using Akka.Actor.Setup;
+
+namespace Akka.Hosting.Coordination
+{
+    public abstract class LeaseOptionBase : IHoconOption
```

Comment:
Historically, `Lease` is part of `Akka.Coordination` which is outside of `Akka.Cluster`. But since we don't have `Akka.Coordination.Hosting`, the closest we can do is to move it to `Akka.Hosting`.

I guess this is just future proving the feature so that it is not tied to clustering in general.

Reply:
But is there anything outside of clustering that would need a lease? Even though they're separate libraries, aren't the clustering components really the only consumers of this?